src/main/java/org/spongepowered/api/world/extent/Extent.java

```diff
@@ -106,81 +105,77 @@
     }
 
     /**
-     * Sets the block at the given position in the world with the provided
-     * {@link Cause} will be used for any events thrown. Note that the
-     * difference between this an {@link MutableBlockVolume#setBlock(Vector3i, BlockState, Cause)} is
-     * that no block tracking chaining will take place. Note that there is
-     * a requirement that the {@link PluginContainer} of the plugin calling
-     * this method is <strong>REQUIRED</strong>.
+     * Sets the block at the given position in the world. Note that the
```

Comment:
Container should already be in the stack...can we just remove the notation on this?